### ISSUE TITLE
Exists a method for...

### ISSUE THREAD

**Issue by soyjavi** [nlp-compromise/nlp_compromise#215]
Hi,

I'm using your **great** library in my project [pet-project](https://github.com/ava-ia/core) and my question ... exists a method for *transpile* a verb like `raining` to a noun `rain`?

**Comment by spencermountain** [comment]
hi Javi! thanks, that looks like a neat project. I like how you've structured things with the processors. Very cool stuff.
i like this idea and think we should implement it. [take a look at](https://github.com/nlp-compromise/nlp_compromise/blob/master/src/data/verbs.js#L90) some random verbs, and it seems likely that only a small amount of verbs would have a corresponding noun like that, dunno.
what should we call it?
thanks!

**Comment by soyjavi** [comment]
Glad to read you like my structure 👍  ... By the way, maybe if we call `root` method with a parameter like `{convertToNoun: true}` or whatever. What do you think?

best,

**Comment by spencermountain** [comment]
yeah, with the rain example, are you connecting `raining` to `rain` for some kind of task understanding? 
just looking at some examples:
* measure -> measurement
* complain -> complaint
* pour -> pour
* proclaim -> proclamation

i suppose it's two-way also, like `Verb.to_noun()` and `Noun.to_verb()`..

**Comment by soyjavi** [comment]
Yeah!, that's ... I'm building a `agnostic assistant core`, which use `intents` for understand a sentence. An `intent` is just a group of rules which have to intersect with the result of compromise (and others). So, is easier intersect one element like `rain` than with all the combinations/conjugations `raining, rains...`.

Now, I don't have too much time for help you in this project... but I will :)

**Comment by soyjavi** [comment]
Any advance with this??
